src/builder.rs

```diff
@@ -347,6 +347,60 @@ impl<'ctx> Builder<'ctx> {
         PointerValue::new(value)
     }
 
+    /// Alignment arguments are specified in bytes, and should always be a power of 2.
+    ///
+    /// The final argument should be a pointer-sized integer.
+    ///
+    /// [`TargetData::ptr_sized_int_type_in_context`](https://thedan64.github.io/inkwell/inkwell/targets/struct.TargetData.html#method.ptr_sized_int_type_in_context) will get you one of those.
+    pub fn build_memcpy<T: IntMathValue<'ctx>>(
+        &self,
+        dest: PointerValue<'ctx>,
+        dest_align_bytes: u32,
+        src: PointerValue<'ctx>,
+        src_align_bytes: u32,
+        size: T,
```

Comment:
It's not clear to me why `size` is an `IntMathValue`. Shouldn't it always just be an int(`IntValue`)? Same for memmove.

Reply:
Seems reasonable! :+1: Changed to `IntValue`.